tests/index.html

```diff
@@ -24,24 +25,23 @@
         along with Kiwix (file LICENSE-GPLv3.txt).  If not, see <http://www.gnu.org/licenses/>
         -->
 
-        <link rel="stylesheet" href="tests/qunit-2.3.2.css" />
-        <script src="tests/qunit-2.3.2.js"></script>
+        <link rel="stylesheet" href="node_modules/qunit/qunit/qunit.css" />
+        <script src="node_modules/qunit/qunit/qunit.js"></script>
+        <script>
+            QUnit.config.autostart = false;
+        </script>
 
         <!-- Using require.js, a module system for javascript, include the
              js files. This loads "main.js", which in turn can load other
              files, all handled by require.js:
              http://requirejs.org/docs/api.html#jsfiles -->
-        <script type="text/javascript" src="www/js/lib/xzdec.js"></script>
-        <script type="text/javascript"
-                data-main="tests/init.js"
-        src="www/js/lib/require.js"></script>
-
-
+        <script src="www/js/lib/xzdec.js"></script>
+        <script src="www/js/lib/require.js" data-main="tests/init.js"></script>
```

Comment:
@Krinkle Could you kindly clarify (for my sake!) whether it's necessary to load RequireJS this way and have it also in the package.json as a dependency? (I don't have much experience with npm repos yet, and exactly how npm loads dependencies: would that require a future webpack config?)

Reply:
The test HTML file is for when you want to run the tests manually in a browser. You'll see the QUnit interface, and can e.g. set `debugger;` breakpoints in the code and step through code and understand failures.

This file is not used by Karma. I should have updated this to load requirejs from node_modules, so that it is identical to Karma.